torch/_utils_internal.py

```diff
@@ -155,6 +155,98 @@ def export_api_rollout_check() -> bool:
     return False
 
 
+class JustknobsConfig:
+    """Represents a lazily loaded config
+
+    This is designed to be used to specify a value in a config.
+
+    i.e. foo.bar = JustknobsConfig("//foo:bar", env_name="FORCE_FOO_BAR")
+
+    This will lazily evaluate it's configuration when compared
+    i.e. if foo.bar:
+
+    It can also be assigned to - i.e.
+    foo.bar = JustknobsConfig("//foo:bar")
+    foo.bar = True
+
+    However, this will force it to always be the overriden value. There isn't a easy way to fix this within
+    python, given how _config.py is generally used.
+
+    Note that this does allow for no JK name (so that you can use this to replace old configurations).
+    """
+
+    def __init__(self, name: Optional[str] = None, env_name=None, default: bool = True):
+        self.name = name
+        self.env_name = env_name
+        self.default = default
+
+    def __bool__(self):
+        return justknobs_feature(
+            self.name, env_name=self.env_name, default=self.default
+        )
```

Comment:
Returning an overloaded bool object makes me a little nervous but I can't actually think of a situation where this would cause a problem (besides if we're doing something dumb like `config is True` which we really shouldn't be doing

Reply:
Yeah playing with this I couldn't even use it correctly reliably, so I'm undoing it + just using a explicit get and set call.